crates/red_knot_python_semantic/src/types/display.rs

```diff
@@ -355,8 +385,40 @@ impl Display for DisplaySpecialization<'_> {
     }
 }
 
+impl<'db> CallableType<'db> {
+    pub(crate) fn display(&'db self, db: &'db dyn Db) -> DisplayCallableType<'db> {
+        DisplayCallableType {
+            signatures: self.signatures(db),
+            db,
+        }
+    }
+}
+
+pub(crate) struct DisplayCallableType<'db> {
+    signatures: &'db [Signature<'db>],
+    db: &'db dyn Db,
+}
+
+impl Display for DisplayCallableType<'_> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        match self.signatures {
+            [signature] => write!(f, "{}", signature.display(self.db)),
+            signatures => {
+                // TODO: How to display overloads?
+                f.write_char('(')?;
```

Comment:
Not sure why (for now) we don't use the same `Overload[...]` syntax used above here as well?

Reply:
Yeah, I think I just missed updating the ones on `CallableType`.